buidl-labs/crypto-code-school-inside-tezos
# claim-bot interaction fixed

Closes #246 

### How to manually test?
1. Go to `/tezos/customizebot`
2. Click on "Claim Bot." It'll take you to `/claim-transaction` page.
3. On the `/claim-transaction` page, click on "Confirm."
